.github/workflows/merge-queue.yml

```
@@ -13,188 +18,147 @@ concurrency:
+  prepare:
+    runs-on: arc-runners
+    timeout-minutes: 35
+    env:
+      AFFECTED_ALL: ${{ secrets.AFFECTED_ALL }}
+      SERVERSIDE_FEATURES_ON: ''
+      DOCKER_REGISTRY: 821090935708.dkr.ecr.eu-west-1.amazonaws.com/
+
+    outputs:
+      TEST_CHUNKS: ${{ steps.build_info.outputs.test_chunkS }}
+      E2E_CHUNKS: ${{ steps.build_info.outputs.e2e_chunkS }}
+      E2E_BUILD_ID: ${{ steps.build_info.outputs.build_iD }}
```

actionlint (1.7.4)

property "build_id" is not defined in object type {build_chunks: string; cache_key: string; e2e_build_id: string; e2e_chunks: string; nx_base: string; random-number: string; test_chunks: string; yarn_cache_folder: string}

(expression)